docs/streams/developer-guide/processor-api.html

```diff
@@ -326,14 +326,14 @@ <h2>
                     <p class="last">If the changelog is disabled then the attached state store is no longer fault tolerant and it can&#8217;t have any <a class="reference internal" href="config-streams.html#streams-developer-guide-standby-replicas"><span class="std std-ref">standby replicas</span></a>.</p>
                 </div>
                 <p>Here is an example for enabling fault tolerance, with additional changelog-topic configuration:
-                    You can add any log config from <a class="reference external" href="https://github.com/apache/kafka/blob/1.0/core/src/main/scala/kafka/log/LogConfig.scala#L61">kafka.log.LogConfig</a>.
+                    You can add any log config from <a class="reference external" href="https://github.com/apache/kafka/blob/1.0/core/src/main/scala/kafka/log/LogConfig.scala#L103">kafka.log.LogConfig</a>.
```

Comment:
If we're going to reference a specific line, we should also reference a specific commit hash, otherwise the same problem will just crop up again next time someone changes the file.

Reply:
Alternatively, we could _not_ reference the line and just point to the file.

Reply:
@vvcephei, since I'm not sure about proper pointer inside the file and considering the fact that link to the line or line+commit can be quickly became outdated, I'm replacing it with file reference as you suggested as alternative.